### PR TITLE
Merge Makefile refactor

### PR DESCRIPTION
**Pull request by AlexanderWinterLLNL**

### CODE REVIEW

File: merlin/config/broker.py [hunk 1]
```diff
@@ -139,7 +139,9 @@ def get_redissock_connection(config_path, include_password):
     return REDISSOCK_CONNECTION.format(**redis_config)
 
 
-def get_redis_connection(config_path, include_password, ssl=False):
+# flake8 complains this function is too complex, we don't gain much nesting any of this as a separate function,
```
Comment (lucpeterson) [inline merlin/config/broker.py:142]: a lot ofd the complexity could be reduced if the CONFIG.broker object  had default values set there

**Review by lucpeterson — APPROVED** [review]

File: merlin/config/broker.py [hunk 2]
```diff
@@ -215,7 +217,13 @@ def get_connection_string(include_password=True):
 
     if broker not in BROKERS:
         raise ValueError(f"Error: {broker} is not a supported broker.")
+    else:
+        return _sort_valid_broker(broker, config_path, include_password)
 
+    return None
```
[the remark this answers is not in the document]
Comment (ben-bay) [reply merlin/config/broker.py:223]: Yeah `None` is the default return in python

**Comment by ben-bay** [comment]
Until we add `config.mk`, all the Makefile targets will be broken.

**Review by ben-bay — CHANGES_REQUESTED** [review]
Until we add `config.mk`, all the `Makefile` targets will be broken on here and `develop`.

**Pull request merged**